Probesys/bileto
# imp: Track and display the last activity of tickets

## Related issue(s)



https://github.com/Probesys/bileto/issues/240
https://github.com/Probesys/bileto/issues/241

## Changes



- Extract the getters and setters for createdAt and createdBy in a trait
- Add the updatedAt and updatedBy fields to the Meta fields (and create the migration)
- Set the updated* fields on persist and update
- Don't track the updated* fields in EntityEvent activity (would be duplicated)
- Set updated* fields of Ticket when posting a message
- Display the last activity date instead of creation in the tickets lists if the two dates are different

## How to test manually



- go to a ticket, make some changes
- go to the tickets list → check the last activity date of the ticket is appearing

## Checklist
